File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.js
function solution(n) {
    var result = 0;
    
    for (var i = 0; i * i <= n; i++) {
      if (n % i === 0) {
        result += (n/i) == i ? 1 : 2;
      }
    }
    
    return result;
}

File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        
        for (int i = 1; i * i <= n; i++) {
            if (n % i == 0) {
                answer += (n / i != i) ? 2 : 1;
            }
        }
        return answer;
    }
}